src/index.scss

```diff
@@ -1,9 +1,19 @@
-$feature-flags: (
```

Comment:
These feature flags are required to make the page lay out correctly, for example in your [deployed application](http://carbon-tutorial-alexg-cambridge.mybluemix.net/#/) the image on the home page is below the "What is carbon?" description

Reply:
this should be done now